src/v2i-hub/CDA1TenthPlugin/test/MobilityOperationConverterTest.cpp

```diff
@@ -0,0 +1,33 @@
+/**
+ * Copyright (C) 2025 LEIDOS.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+#include <gtest/gtest.h>
+#include <tmx/j2735_messages/J2735MessageFactory.hpp>
+#include "MobilityOperationConverter.h"
+
+
+TEST(MobilityOperationConverterTest, fromTree)
```

Comment:
This unit test does not check anything. Creating unit tests that have code coverage but don't actually check anything provides missing leading data about how much of our code is actually being evaluated by unit test. Please either remove this unit test or EXPECT some outputs